Dockerfile

```diff
@@ -49,4 +35,5 @@ COPY --from=build-env /app/build/web /usr/share/nginx/html
 EXPOSE 80
 
 # Before stating NGinx, re-zip all the content to ensure customizations are propagated
-CMD gzip -k -r /usr/share/nginx/html/ && nginx -g 'daemon off;'
+RUN gzip -k -f -r /usr/share/nginx/html/ 
+CMD ["nginx", "-g", "daemon off;"]
```

Comment:
No this was intendeed to rezip assets upon container startup. So that you can override whatever you want. The corresponding zip would be the right one.

Reply:
No I mean **don't change this line**.

Reply:
If you run gzip every time the container starts, it will fail if the container restarts cause gzip will ask if you want to overwrite the existing files.

Reply:
Ok, true. Then please modify the gzip command to overwrite the existing zipped fail.

But keep the gzip on the CMD line.

Reply:
We can't do that either, cause the `-f` flag will recompress the already compressed files.

Reply:
It's higher priority to me to get the right zipped content.

Not being able to restart the container is less impacting to me.

So please either you make a constructive proposal addressing my concerns, or you revert to the previous value.

Reply:
How about putting things in one line and swallowing the errors upon gzip step?

Reply:
I'll just revert to the previous CMD then.